object/adapter.go

```diff
@@ -30,17 +28,18 @@ type Adapter struct {
 	Name        string `xorm:"varchar(100) notnull pk" json:"name"`
 	CreatedTime string `xorm:"varchar(100)" json:"createdTime"`
 
-	Type            string `xorm:"varchar(100)" json:"type"`
-	DatabaseType    string `xorm:"varchar(100)" json:"databaseType"`
+	Type          string `xorm:"varchar(100)" json:"type"`
+	SameAsCasdoor bool   `xorm:"bool" json:"sameAsCasdoor"`
```

Comment:
For security issue, it is necessary use field `SameAsCasdoor` to avoid storing host, password of adapter that same as Casdoor itself.